internal/crdao/services/event_tracking/track_transfer.go

```diff
@@ -0,0 +1,54 @@
+package event_tracking
+
+import (
+	"time"
+
+	"casper-dao-middleware/internal/crdao/di"
+	"casper-dao-middleware/internal/crdao/entities"
+	"casper-dao-middleware/internal/crdao/events"
+	"casper-dao-middleware/pkg/casper"
+	"casper-dao-middleware/pkg/go-ces-parser"
+)
+
+type TrackTransfer struct {
+	di.EntityManagerAware
+	di.DAOContractsMetadataAware
+
+	deployProcessed casper.DeployProcessed
+	cesEvent        ces.Event
+}
+
+func NewTrackTransfer() *TrackTransfer {
+	return &TrackTransfer{}
+}
+
+func (s *TrackTransfer) SetCESEvent(event ces.Event) {
+	s.cesEvent = event
+}
+
+func (s *TrackTransfer) SetDeployProcessed(deployProcessed casper.DeployProcessed) {
```

Comment:
Let's move into `di` traits?

Additionally, what is `deployProcessed`? Is it short for `DeployProcessedEvent` or is it `ProcessedDeploy`? I think we can improve naming here.